account_financial_report/report/journal_ledger.py

```diff
@@ -100,9 +101,9 @@ def _get_move_lines_data(self, ml, wizard, ml_taxes, auto_sequence, exigible):
             base_debit = ml_taxes and ml.debit or 0.0
             base_credit = ml_taxes and ml.credit or 0.0
             base_balance = ml_taxes and ml.balance or 0.0
-            tax_debit = ml.tax_line_id and ml.debit or 0.0
-            tax_credit = ml.tax_line_id and ml.credit or 0.0
-            tax_balance = ml.tax_line_id and ml.balance or 0.0
+            tax_debit = ml.tax_line_id.id and ml.debit or 0.0
```

Comment:
Why is this better?

Reply:
Tested and its the same in terms of performance, so I rolled back these changes to keep them as they were before.